harvard-nus-geography.csl

```diff
@@ -0,0 +1,309 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Harvard-NUS-Geography</title>
+    <title-short>Citation style for Geography Honours Thesis in National University of Singapore.</title-short>
+    <id>http://www.zotero.org/styles/harvard-nus-geography</id>
+    <link href="http://www.zotero.org/styles/harvard-nus-geography" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-cite-them-right" rel="template"/>
+    <link href="http://www.fas.nus.edu.sg/geog/documents/undergraduate/HT_Guide_JG_2017.pdf" rel="documentation"/>
+    <author>
+      <name>Kin Guan Wee</name>
+      <email>slbwkg@nus.edu.sg</email>
+      <uri>http://www.mendeley.com/profiles/kin-guan-wee2/</uri>
+    </author>
+    <contributor>
+      <name>Scott Wagstaff</name>
+      <uri>http://www.mendeley.com/profiles/scott-wagstaff/</uri>
+    </contributor>
+    <contributor>
+      <name>Kin Guan Wee</name>
+      <uri>http://www.mendeley.com/profiles/kin-guan-wee2/</uri>
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="geography"/>
+    <updated>2017-10-26T09:54:23+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  
+  <locale xml:lang="en-GB">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds</multiple>
+      </term>
+      <term name="editortranslator" form="verb">edited and translated by</term>
+      <term name="edition" form="short">ed.</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <choose>
+      <if type="chapter paper-conference" match="any">
+        <names variable="container-author" delimiter=", " suffix=", ">
+          <name and="text" initialize-with=". " delimiter=", " sort-separator=", " name-as-sort-order="all"/>
```

Comment:
names should be with `and="symbol"` throughout. `name-as-sort-order="all"` is wrong here.

Reply:
Ditto the and="symbol". M

May I check with you why the `name-as-sort-order="all"` should be changed? All authors' names ought to be displayed as *lastname-firstname*.